packages/intl-localematcher/abstract/LookupSupportedLocales.ts

```diff
@@ -21,7 +21,8 @@ export function LookupSupportedLocales(
       noExtensionLocale
     )
     if (availableLocale) {
-      subset.push(availableLocale)
+      // test262/test/intl402/DurationFormat/supportedLocalesOf/locales-specific.js
```

Comment:
This seems like a pre-existing bug, we're returning the resolved locales not the inputted ones.